src/User/Application/DTO/ConfirmPasswordResetDto.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\User\Application\DTO;
+
+use Symfony\Component\Serializer\Annotation\Groups;
+
+final readonly class ConfirmPasswordResetDto
+{
+    public function __construct(
+        #[\SensitiveParameter]
+        #[Groups(['confirm_password_reset:write'])]
```

Comment:
please remove serialization groups from this file  since we can expose all of them to the api user by default